Neos.ContentRepository.Core/Classes/CommandHandler/CommandBus.php

```diff
@@ -17,16 +17,19 @@
 final class CommandBus
 {
     /**
-     * @var CommandHandlerInterface[]
+     * @var (CommandHandlerInterface|ControlFlowAwareCommandHandlerInterface)[]
```

Comment:
IMO we don't need the additional interface because we already check for the return type in `ContentRepository::handle()`